instana/resource-sli-config.go

```diff
@@ -0,0 +1,261 @@
+				SliConfigFieldMetricThreshold: {
+					Type:        schema.TypeFloat,
+					Required:    true,
+					Description: "The threshold for the metric configuration",
+					ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+						v := val.(float64)
+						if v <= 0 {
+							errs = append(errs, fmt.Errorf("Metric threshold must be higher than 0"))
+						}
+						return
+					},
+				},
+			},
+		},
+	}
+
+	//SliConfigSliEntity schema field definition of instana_sli_config field sli_entity
+	SliConfigSliEntity = &schema.Schema{
+		Type:        schema.TypeList,
+		Required:    true,
+		Description: "The entity to use for the SLI config",
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				//SliConfigFieldSliType nested schema field definition of instana_sli_config field sli_entity.sli_type
+				SliConfigFieldSliType: {
+					Type:         schema.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringInSlice([]string{"application", "custom", "availability"}, true),
+					Description:  "The entity type (application, custom, availability)",
+				},
+				//SliConfigFieldApplicationId nested schema field definition of instana_sli_config field sli_entity.application_id
+				SliConfigFieldApplicationID: {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "The application ID of the entity",
+				},
+				//SliConfigFieldServiceId nested schema field definition of instana_sli_config field sli_entity.service_id
+				SliConfigFieldServiceID: {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "The service ID of the entity",
+				},
+				//SliConfigFieldEndpointId nested schema field definition of instana_sli_config field sli_entity.endpoint_id
+				SliConfigFieldEndpointID: {
+					Type:        schema.TypeString,
+					Optional:    true,
+					Description: "The endpoint ID of the entity",
+				},
+				//SliConfigFieldBoundaryScope nested schema field definition of instana_sli_config field sli_entity.boundary_scope
+				SliConfigFieldBoundaryScope: {
+					Type:         schema.TypeString,
+					Required:     true,
+					ValidateFunc: validation.StringInSlice([]string{"ALL", "INBOUND"}, true),
+					Description:  "The boundary scope for the entity configuration (ALL, INBOUND)",
+				},
+			},
+		},
+	}
+)
+
+//NewSliConfigResourceHandle creates the resource handle for SLI configuration
+func NewSliConfigResourceHandle() *ResourceHandle {
+	return &ResourceHandle{
+		ResourceName: ResourceInstanaSliConfig,
+		Schema: map[string]*schema.Schema{
+			SliConfigFieldName:                       SliConfigName,
+			SliConfigFieldFullName:                   SliConfigFullName,
+			SliConfigFieldInitialEvaluationTimestamp: SliConfigInitialEvaluationTimestamp,
+			SliConfigFieldMetricConfiguration:        SliConfigMetricConfiguration,
+			SliConfigFieldSliEntity:                  SliConfigSliEntity,
+		},
+		SchemaVersion: 1,
```

Comment:
The initial schema version should be 0

Reply:
misconception on my part, initial schema version set to 0